### DISCUSSION TITLE
How to add realms' roles to the admin role on the master realm ?

### DISCUSSION THREAD

**Discussion by kadhemmejri** [keycloak/keycloak#9412] [Q&A]
Hello,
I removed all the roles affected to a realm from the role admin in the realm "master"
Now I want to add them again, I always get the message "forbidden". 
My question : is there any way to add them again ?

More details ::
on my keycloak I have many realms.
I went to domain roles in the "master" domain
I deselected the roles 
![image](https://user-images.githubusercontent.com/70603515/148402374-e8a52e36-2092-4b93-95f7-3761f1ec70ba.png)
Now I want to add them again, I cannot. When I select them and click "add selected" I get the message forbidden

![image](https://user-images.githubusercontent.com/70603515/148402580-ba26a028-c6fe-42c3-a814-53bbbd09cb9b.png)

PLEASE, THE NEED IS VERY URGENT. HELP ME ASAP

**Comment by bs-matil** [comment]
You can may do this in the database. Manually. No other ideas sorry. Also this is not a support line but a community forum ;)
